modules/blockuser.cpp

```diff
@@ -126,6 +133,7 @@ class CBlockUser : public CModule {
             PutModule("This user is not blocked");
     }
 
+    //Compressing all above functions into one for embedded web requests.
```

Comment:
What i'm trying to say is this that this gives the user the same functionality as the other commands, not sure how to word it or if it is necessary.